src/blocks.js

```diff
@@ -174,9 +174,9 @@ export default class CodeMirrorBlocks {
     }
   }
 
-  markText(from, to, options) {
-    function poscmp(a, b) { return a.line - b.line || a.ch - b.ch; }
+  poscmp(a, b) { return a.line - b.line || a.ch - b.ch; }
```

Comment:
`poscmp` can probably be a module function instead of an instance method. Also add a doc comment about what the behavior is. I always get so confused with comparator functions and I can never remember what negative or positive return values actually mean.